Packs/ApiModules/Scripts/DemistoClassApiModule/DemistoClassApiModule.py

```diff
@@ -1,28 +1,119 @@
 import demistomock as demisto  # noqa: F401
+import json
 import sys
 
+from datetime import datetime
+
 if sys.version_info[0] >= 3:
     try:
         class DemistoWrapper(Demisto):  # type:ignore  [name-defined] # noqa: F821 # pylint: disable=E0602
             """A content-side wrapper to the builtin Demisto class.
             All methods of this class can be executed in both scripts and integrations
             (E.g., self.results). """
 
+            def initialize(self):
```

Comment:
We use this to mark internal methods right?
```suggestion
            def _initialize(self):
```

Reply:
yes but this is called from outside (see bottom of the ApiModule)